bridges/opentracing/lib/opentelemetry/bridge/opentracing/tracer.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Bridge
+    module OpenTracing
+      # Tracer provides a means of referencing
+      # an OpenTelemetry::Tracer as a OpenTracing::Tracer
+      class Tracer
+        HTTP_TEXT_FORMAT = OpenTelemetry::DistributedContext::Propagation::HTTPTextFormat.new
+        BINARY_FORMAT = OpenTelemetry::DistributedContext::Propagation::BinaryFormat.new
+
+        def active_span
+          OpenTelemetry.tracer.current_span
+        end
+
+        def start_active_span(operation_name,
+                              child_of: nil,
+                              references: nil,
+                              start_time: Time.now,
+                              tags: nil,
+                              ignore_active_scope: false,
+                              finish_on_close: true,
+                              &block)
+          span = OpenTelemetry.tracer.start_span(operation_name,
+                                                 with_parent: child_of,
+                                                 attributes: tags,
+                                                 links: references,
+                                                 start_timestamp: start_time)
+          OpenTelemetry.tracer.with_span(span, &block) if block_given?
+          Scope.new(ScopeManager.new, span, finish_on_close)
+        end
+
+        def start_span(operation_name,
+                       child_of: nil,
+                       references: nil,
+                       start_time: Time.now,
+                       tags: nil,
+                       ignore_active_scope: false,
+                       &block)
+          span = OpenTelemetry.tracer.start_span(operation_name,
+                                                 with_parent: child_of,
+                                                 attributes: tags,
+                                                 links: references,
+                                                 start_timestamp: start_time)
+          OpenTelemetry.tracer.with_span(span, &block) if block_given?
```

Comment:
Similar to `start_active_span`, when a block is passed to `start_span` it will yield the span and return the block's return value.